snap/local/runtime-helpers/bin/security-secretstore-setup-launch.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -ex
+
+security-secretstore-setup -confdir "$SNAP_DATA/config/security-secretstore-setup/res" --vaultInterval=10
+
+if [ -n "$REDIS5_PASSWORD_PATHNAME" ]; then
+
+    CONFIG_FILE="$SNAP_DATA/config/security-secretstore-read/res/configuration.toml"
+    # grep -q returns 0 if a match is found
+    grep -q "/tmp/edgex" "$CONFIG_FILE"
+    RETVAL="$?"
+
+    if [ $RETVAL == 0 ]; then
```

Comment:
why isn't this whole thing just 

```suggestion
    if grep -q "/tmp/edgex" "$CONFIG_FILE"; then
```

Reply:
Cause my shell fu isn't as strong as yours... thanks for the suggestion!